cogs/github.py

```diff
@@ -0,0 +1,101 @@
+from discord import Embed, Color
+from discord.ext.commands import Cog, Context, command, Bot, cooldown, BucketType
+from .utils.constants import Emojis
+import aiohttp
+
+BAD_RESPONSES = {
+    404: "Issue/pull request not Found! Please enter a valid PR Number!",
+    403: "Rate limit is hit! Please try again later!"
```

Comment:
403 is Forbidden
429 is too Much Requests
I think you got the wrong code

Reply:
it's according to the github api

Reply:
Really ? Then the GitHub API is really strange, not using the real codes...